### DISCUSSION TITLE
My Registry has been submitted for a week, when will it be approved?

### DISCUSSION THREAD

**Discussion by superjakc** [WalletConnect/walletconnect-monorepo#850] [Developer Support]
What information do I need to provide? Thank you for your guidance!

**Comment by chadyj** [comment]
What is the name of your wallet or dapp?  I'll check it out.

**Reply by superjakc** [reply]
Name:DDC. Please check

**Comment by superjakc** [comment]
DDCWallte

subrahmanyam posa ***@***.***> 于2022年3月11日周五 19:44写道：
